filtered-resource-service/src/unit-tests/java/uk/gov/gchq/palisade/service/filteredresource/model/MessageSerialisationTest.java

```diff
@@ -40,20 +40,30 @@
                     Arguments.of(ApplicationTestData.REQUEST),
                     Arguments.of(AuditSuccessMessage.Builder.create(
                             ApplicationTestData.REQUEST,
-                            Collections.emptyMap())
-                            .withLeafResourceId(ApplicationTestData.RESOURCE_ID)),
+                            Collections.emptyMap())),
```

Comment:
Is there a reason you've removed the leafResourceId from the AuditSuccessMessage?

Reply:
Its automatically gathered from the `.create(Request, ...)` method, using the request's `.getResource().getId()`